code/datums/storage/storage.dm

```diff
@@ -962,7 +962,7 @@ GLOBAL_LIST_EMPTY(cached_storage_typecaches)
 		show_contents(to_show)
 		return FALSE
 
-	if(!isliving(to_show) || !to_show.can_perform_action(parent))
+	if(!isliving(to_show) || !to_show.can_perform_action(parent, ALLOW_RESTING | NEED_HANDS | FORBID_TELEKINESIS_REACH))
```

Comment:
The `needs_hands` check is going to break silicons opening bags I think?

Reply:
Which is/was 100% possible prior

Reply:
Sure I'll remove it